styledes.css

```diff
@@ -0,0 +1,599 @@
+  }
+
+  #dropdown_header {
+    width: 100%;
+    height: 72px;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .dropdown {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    padding: 12px;
+    left: 0;
+    gap: 12px;
+    width: 75%;
+    height: 120px;
+  }
+
+  #java_dropdown {
+    width: 122px;
+    height: 120px;
+    background: #f7f7f9;
+    border-radius: 8px;
+    flex: none;
+    order: 0;
+  }
+
+  .drp_txt {
+    position: relative;
+    width: 98px;
+    height: 24px;
+    left: 8px;
+    top: 50px;
+  }
+
+  .separate {
+    position: absolute;
+    height: 1;
+    left: 0;
+    right: 0;
+    bottom: 0;
+  }
+
+  .hr {
+    position: absolute;
+    left: 0;
+    right: 0;
+    bottom: 0;
+  }
+
+  .contactMe {
+    width: 100%;
+    height: 846px;
+    left: 0;
+    top: 4110px;
+    background-image: url(./asset/Contact/contactdesk.svg);
+    background-repeat: no-repeat;
+    background-size: 95% 95%;
+    background-position: 90% 80%;
+  }
+
+  .contactblock {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    align-content: center;
+    padding: 0;
+    gap: 20px;
+    position: absolute;
+    width: 57%;
+    height: 577px;
+    margin-top: 125px;
+    margin-left: 320px;
+  }
+
+  .contactTxt {
+    width: 100%;
+    height: 56px;
+    position: absolute;
+    margin-top: -55px;
+    text-align: center;
+    flex: 0;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .Contact_body_text {
+    align-content: center;
+    width: 100%;
+    height: auto;
+    left: 0;
+    top: 80px;
+    flex: 0;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  .body-text {
+    margin-left: 105px;
+    justify-content: center;
+    align-content: center;
+    width: 70%;
+    top: 0;
+    height: fit-content;
+  }
+
+  .form {
+    display: flex;
+    flex-direction: column;
+    margin-top: 250px;
+    width: 100%;
+    height: auto;
+    margin-left: 24px;
+    margin-right: 24px;
+  }
+
+  #name {
+    border-radius: 8px;
+    border: 1px #cfd8dc solid;
+    background-color: #fff;
+    margin-bottom: 20px;
+    margin-left: 150px;
+    margin-right: 150px;
+    padding: 10px 50px;
+    width: auto;
+  }
+
+  #mail {
+    border-radius: 8px;
+    border: 1px #cfd8dc solid;
+    background-color: #fff;
+    margin-left: 150px;
+    margin-right: 150px;
+    margin-bottom: 15px;
+    padding: 10px 50px;
+    width: auto;
+  }
+
+  #textarea {
+    border-radius: 8px;
+    border: 1px #cfd8dc solid;
+    background-color: #fff;
+    margin-left: 150px;
+    margin-right: 150px;
+    margin-bottom: 15px;
+    padding: 10px 50px;
+    width: auto;
+  }
+
+  .normalButtnAbout {
+    display: flex;
+    width: 175px;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #396df2;
+    font-size: 17px;
+    font-weight: 500;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    left: 300px;
+    cursor: pointer;
+    background-color: #fff;
+  }
```

Comment:
- [ ] The `get in touch` button is not exactly centered and at some other breakpoint it's not centered at all, so please ensure that the button is always centered for the desktop screens👍🏼 

### Your Design
![image](https://user-images.githubusercontent.com/68030297/221280669-402068d2-b316-48b1-a2cb-56551d541f4d.png)

![image](https://user-images.githubusercontent.com/68030297/221280862-a3ffb473-4208-4f12-b47d-4028f490310b.png)

### Figma Design
![image](https://user-images.githubusercontent.com/68030297/221281045-00f1bdb8-3256-45b4-99d6-3a692116d35e.png)